code/datums/supplypacks.dm

```diff
@@ -728,6 +728,110 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	cost = 150
 	containertype = /obj/structure/closet/crate/basic
 	containername = "contacts crate"
+	group = "Clothing"	
+	
+//Winter Coats//
+	
+	/datum/supply_packs/engwinter
```

Comment:
All of this is indented for some reason
It shouldn't be
Move everything 1tab backwards